docs/index.adoc

```diff
@@ -12,33 +11,61 @@ This documentation is available in {pages}[HTML] and {pages}/ebook.pdf[PDF] form
 Software is available from the {project}[Korrel8r project].
 ====
 
-include::description.adoc[]
+[id="index-overview"]
+== Korrel8r overview
+
```

Comment:
This overview is a thing of beauty. I could not have said it better myself, and believe me I tried.

Reply:
Haha you basically did, it's pretty much your content slightly rewritten, tweaked, and reorganized 😉 glad you like it though!